django-stubs/contrib/auth/models.pyi

```diff
@@ -102,33 +103,34 @@ class AbstractUser(AbstractBaseUser, PermissionsMixin):
     def get_full_name(self) -> str: ...
     def get_short_name(self) -> str: ...
     def email_user(
-        self, subject: str, message: str, from_email: str = ..., **kwargs: Any
+        self, subject: str, message: str, from_email: str | None = ..., **kwargs: Any
     ) -> None: ...
 
-class User(AbstractUser):
```

Comment:
is this meant to be deleted?

Reply:
It's not deleted, it's just moved down to line 111

Reply:
oh whoops! thank you!